vue-gtag.d.ts

```diff
@@ -0,0 +1,73 @@
+declare module 'vue-gtag' {
+  import _Vue, { PluginFunction } from 'vue';
+
+  export interface PageView {
+    page_path: string;
+  }
+
+  export interface Event {
+    event_category: string;
+    event_label: string;
+    value: string;
+  }
+
+  export interface ScreenView {
+    app_name: string;
+    screen_name: string;
+  }
+
+  export interface Purchase {
+    transaction_id: string;
+    affiliation: string;
+    value: number;
+  }
+
+  export interface Linker {
+    domains: string[];
+  }
+
+  export interface Exception {
+    description: string;
+    fatal: boolean;
+  }
+
+  export type Dictionary<T> = { [key: string]: T };
+
+  export interface VueGtag {
+    pageview(pageView: PageView): void;
+    event(action: string, event: Event): void;
+    screenview(screenView: ScreenView): void;
+    customMap(map: Dictionary<string>): void;
+    purchase(purchase: Purchase): void;
+    linker(config: Linker): void;
+    exception(ex: Exception): void;
+  }
+
+  export interface PluginOptions {
+    pageTrackerTemplate?: () => void;
+    onBeforeTrack?: () => void;
+    onAfterTrack?: () => void;
+    onReady?: () => void;
+    enabled?: boolean;
+    disableScriptLoad?: boolean;
+    bootstrap?: boolean;
+    globalObjectName?: string;
+    pageTrackerEnabled?: boolean;
+    pageTrackerScreenviewEnabled: boolean;
```

Comment:
isn't this optional?

Reply:
Do you mean these options are optional? These are optional, that is what the `?` means

Reply:
my assertion: `pageTrackerScreenviewEnabled` is not marked as optional, but it should be

Reply:
Sorry, GitHub comments aren't that clear if you meant those 4 lines or just one and I didn't notice the missing `?`

I've fixed this in the latest commit.